### PR TITLE
kickoff and login design

### PR DESCRIPTION
**Pull request by damfinkel**
## Summary

- Kickoff using ng new
- Removed default snippets from app.component
- Implemented minimum design for login

### CODE REVIEW

File: angular/tech-show-frontend/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/.editorconfig:2]: remove editor config file

File: angular/tech-show-frontend/src/app/app.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="main-container">
+  <login></login>
+</div>
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/app.component.html:3]: missing eol

File: angular/tech-show-frontend/src/app/app.component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="main-container">
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/app.component.html:1]: I dunno if this container is required

File: angular/tech-show-frontend/src/app/app.component.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.main-container {
+  height: 100vh;
+  width: 100vw;
+}
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/app.component.scss:4]: EOL

File: angular/tech-show-frontend/src/app/components/login/login.component.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<div class="login-container">
+  <div class="login-form-container">
+    <h1 class="login-title">Login</h1>
+    <label htmlFor="email" class="login-input-title">Email</label>
+    <input id="email" class="login-input" />
+    <label title="password">Password</label>
+    <input id="email" class="login-input-title" class="login-input" />
+    <button type="button" class="login-button">Login</button>
+  </div>
+</div>
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/components/login/login.component.html:10]: EOL

File: angular/tech-show-frontend/src/app/components/login/login.component.scss [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  max-width: 300px;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.login-form-container {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/components/login/login.component.scss:17]: absolute?

File: angular/tech-show-frontend/src/app/components/login/login.component.scss [hunk 7]
```diff
@@ -0,0 +1,39 @@
+.login-container {
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/components/login/login.component.scss:1]: absolute?

File: angular/tech-show-frontend/src/app/components/login/login.component.scss [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+.login-input {
+  border: 1px solid #F0F0F0;
+  height: 40px;
+  margin-bottom: 20px;
+}
+
+.login-button {
+  align-self: center;
+}
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/app/components/login/login.component.scss:39]: missing eol

File: angular/tech-show-frontend/src/styles.scss [hunk 9]
```diff
@@ -0,0 +1,9 @@
+/* You can add global styles to this file, and also import other style files */
+
+* {
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
```
Comment (FrankIglesias) [inline angular/tech-show-frontend/src/styles.scss:9]: missing eol
install equalizer if you like

File: angular/tech-show-frontend/src/app/components/login/login.component.scss (hunk 6 above)
Comment (damfinkel) [reply angular/tech-show-frontend/src/app/components/login/login.component.scss:17]: yes, so I center vertically the login

File: angular/tech-show-frontend/src/styles.scss (hunk 9 above)
Comment (damfinkel) [reply angular/tech-show-frontend/src/styles.scss:9]: I couldn't decide if installing equalizer or keep it as simple as possible. What do you think?

Comment (FrankIglesias) [reply angular/tech-show-frontend/src/styles.scss]: I'd use the equalizer due to its low installing complexity

**Pull request merged**